score/cli.py

```
@@ -84,5 +88,21 @@ def scrape_pypi_web(start, end, output):
     click.echo("Scraping completed.")
 
 
+@cli.command()
+@click.option(
+    "--letter_to_scrape",
+    "-l",
+    required=True,
+    help="Enter the starting letter or number to scrape (e.g., 'a' or '0').",
+)
+def conda(letter_to_scrape):
+    try:
+        subprocess.run(["python", "score/conda.py"], check=True)
+        scrape_conda_packages(letter_to_scrape)
+        click.echo("Scraping completed.")
+    except subprocess.CalledProcessError as e:
+        click.echo(f"Error: {e}")
```

please don't catch and ignore exceptions